src/Components/Article/ArticlePage.jsx

```diff
@@ -15,7 +14,11 @@ const ArticlePage = () => {
   return (
     <div>
       {articles.map((article, index) => {
-        return <ArticleCard key={index} article={article} />;
+        return (
+          <>
```

Comment:
This fragment isn't needed as you can just return the ArticleCard component in your map.